common/Kconfig

```diff
@@ -964,6 +964,14 @@ config SPL_FIOVB
 	 This option enables compilation of Foundries.IO verified boot
 	 operations
 
+config SCP03
+	bool "Build SCP03 - Secure Channel Protocol O3 - controls"
+	depends on OPTEE
```

Comment:
It depends on both TEE and OPTEE (and on our current branch OPTEE won't force TEE, not sure if it is still the case upstream).

Without TEE it fails to build (tee_ functions missing).

Reply:
I'll check that